### ISSUE TITLE
Cut 0.32.0 Release

### ISSUE THREAD

**Issue by FedeDP** [falcosecurity/falco#1993]
## Release Steps

Process is described in [this document](https://github.com/falcosecurity/falco/blob/master/RELEASE.md).

## Release Blocking PRs

- [ ] [[part 1/n] Introduce vtable-based dispatch](https://github.com/falcosecurity/libs/pull/213)
- [ ] [wip: new(libscap): add parsers for gVisor seccheck messages](https://github.com/falcosecurity/libs/pull/318)
- [ ] [new(userspace/falco): added an option to listen to changes on the config file and rules files, and trigger a Falco reload](https://github.com/falcosecurity/falco/pull/1991)
- [ ] [Include origin host in output json](https://github.com/falcosecurity/falco/pull/1989)
- [ ] [Falco Rules/Conditions update](https://github.com/falcosecurity/falco/pull/1981)
- [ ] [New Rule Detect Linux Cgroup Container Escape Vulnerability (CVE-2022-0492)](https://github.com/falcosecurity/falco/pull/1969)

**Moreover, a new release of Driverkit is needed, because latest release (0.7.0) has a bug that prevents amazonlinux drivers builds.**

## Nice-to-have PRs

- [ ] [ wip: refactor(userspace/libsinsp)!: introducing plugin manager](https://github.com/falcosecurity/libs/pull/313)
- [ ] [update(libsinsp): drop also untracked syscalls in kernel-side simple consumer mode](https://github.com/falcosecurity/libs/pull/211)
- [ ] [new(docker,scripts): port all docker images to be multiarch ready](https://github.com/falcosecurity/falco/pull/1990)
- [ ] [Add job to generate configs from crawled kernels](https://github.com/falcosecurity/test-infra/pull/684)

### Driverkit
- [ ] [new(driverkit): allow configurable builder images](https://github.com/falcosecurity/driverkit/pull/113)
- [ ] [new: arm64 support](https://github.com/falcosecurity/driverkit/pull/143)

## Action Items

- [ ] **Pre-Release**
  - [ ] Pick a final driver version in Falco
  - [ ] Release and publish plugins 
  - [ ] Prebuilt drivers publish 
  - [ ] Milestones: https://github.com/falcosecurity/falco/milestone/22
  - [ ] Changelog
- [ ] **Release**
  - [ ] Tag
  - [ ] Github Release
- [ ] **Docs**
- [ ] **Website**
- [ ] **Announcements**
- [ ] **Post-Release**

## Open Questions

/milestone 0.32.0

/assign @fededp

/kind documentation

**Comment by leogr** [comment]
Hey @FedeDP 

Could you include https://github.com/falcosecurity/falco/issues/1994 too, please? :pray:

**Comment by FedeDP** [comment]
Of course!

**Comment by jasondellaluce** [comment]
Hey @FedeDP, I just opened a PR in `falco-website` to update the plugins documentation with the newest plugin API changes (https://github.com/falcosecurity/falco-website/pull/581). Would you add it to the list of post-release actions? This is also tracked in https://github.com/falcosecurity/falco/issues/1948 as a backup.

**Comment by jasondellaluce** [comment]
@FedeDP, would you add https://github.com/falcosecurity/falco-website/pull/584 to the list? This is the docs migration from the legacy k8s audit implementation to the plugin-based one. This would fit in the post-release actions.

**Comment by FedeDP** [comment]
Yay!

**Comment by leogr** [comment]
Hey @FedeDP 

Since the upcoming release is very close, we don't have enough time to test the ARM support. Should we move it to the next release?

**Comment by FedeDP** [comment]
It's very sad to hear that :(  
But i agree with you, we have no time to thoroughly test this! It would be great to have a patch release for this though, perhaps as soon as we are confident with the changes.

Providing arm64 artifacts for Falco has a huge value!

**Comment by leogr** [comment]
> It would be great to have a patch release for this though, perhaps as soon as we are confident with the changes.

I totally agree. We are very close to achieving official support for ARM64, but we have some stuff to fix and try. I'm ok with releasing a patch release once we fix the pending issues.

**Comment by leogr** [comment]
This release looks very nice :sunglasses:

**Comment by FedeDP** [comment]
It's because of the release manager sir :crown:

**Comment by FedeDP** [comment]
We have to bump driverversion unfortunately as during our tests we spotted some issues.

**Comment by leogr** [comment]
> We have to bump driverversion unfortunately as during our tests we spotted some issues.

Just testing the fix :point_right: https://github.com/falcosecurity/libs/pull/361 :sunglasses:

**Comment by FedeDP** [comment]
We will need to bump driverversion once more given that we found a couple of bugs during our tests :(

**Comment by leogr** [comment]
Quick update regarding plugins.

We performed some manual tests on plugins and found some issues. Because of that, we need to release a new version of the go SDK. Then we will have to release and bump all plugins again.

However, we are confident that we will complete those tasks today. I'll keep you posted.

**Comment by FedeDP** [comment]
New plugins are released and the PR to upgrade them in Falco is opened: https://github.com/falcosecurity/falco/pull/2033
